streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractKeyValueStoreTest.java

```diff
@@ -422,6 +503,21 @@ public void shouldNotThrowInvalidRangeExceptionWithNegativeFromKey() {
                     " Note that the built-in numerical serdes do not follow this for negative numbers")
             );
         }
+    }
+
+    @Test
+    public void shouldNotThrowInvalidReverseRangeExceptionWithNegativeFromKey() {
```

Comment:
Can we add tests for some other invalid range cases? For example with both bounds positive but from > to

Reply:
ack. added.